src/events/order/cd-event-ticket.vue

```diff
@@ -3,8 +3,11 @@
     <div class="cd-event-tickets__head"></div>
     <div class="cd-event-tickets__ticket">
       <span class="cd-event-tickets__name"><span class="cd-event-tickets__name-for">Name:</span>{{ user.firstName }} {{ user.lastName }}</span>
-      <div class="cd-event-tickets__ticket-selector">
-        <multiselect v-model="tickets" :options="ticketsOptions" group-label="name" group-values="tickets" :multiple="true" :searchable="false" :group-select="false" :placeholder="$t('Select tickets')" track-by="id" label="name" @close="onBlur" @open="onFocus" :data-vv-name="`tickets-${user.id}`" v-validate="'required'"></multiselect>
+      <div class="cd-event-tickets__ticket-selector" v-if="!ticketsAreFull(tickets)"> 
+        <multiselect v-model="selectedTickets" :options="ticketsOptions" group-label="name" group-values="tickets" :multiple="true" :searchable="false" :group-select="false" :placeholder="$t('Select tickets')" track-by="id" label="name" @close="onBlur" @open="onFocus" :data-vv-name="`tickets-${user.userId}`" v-validate="'required'"></multiselect>
+      </div>
+      <div v-else>
+        {{ $t('Whoopsies, no more tickets available for you :( ') }}
```

Comment:
This string definitely needs to be changed 😅 

"Sorry, there are no tickets left for this event" or something like that.